README.md

```diff
@@ -17,9 +17,20 @@ Additional documentation:
 [spam-bbs-dev]: https://bbs.spamgourmet.com/viewforum.php?f=2
 [spam-bbs-dev-arch]: https://web.archive.org/web/*/https://bbs.spamgourmet.com/viewforum.php?f=2
 
+### HELPING WITH DEVELOPMENT
+
+See the files in the `docs` directory, especially `dev-notes.md` 
```

Comment:
To improve the odds that casual passers-by who just want to fix one thing will find the relevant information, it may be worth having a `CONTRIBUTING.md` in the root which summarizes the key information and refers to the `docs` for the finer details.

Reply:
I agree; won't add it at this point just to simplify the current PRs, but a PR to add such a document would be a great idea.